File: java/src/model/ataque/PomadaWassington.java
package model.ataque;

import model.chimpos.Chinpokomon;

public class PomadaWassington extends Ataque {

    public PomadaWassington(String nombre) {
        super(5, nombre, 1);
    }

    @Override
    public void atacar(Chinpokomon chipo, Integer valor) {
        chipo.getOponente().agregarVida(valor);
    }

}

File: python/model/ataque/PomadaWassington.py
from model.ataque.Ataque import Ataque

class PomadaWassington(Ataque):
        
        def __init__(self):
            Ataque.__init__(self, 5, "Pomada Wassington", 1)
        
        def atacar_con_valor(self, chipo, valor):
            chipo.oponente.agregar_vida(valor)